cmd/livepeer/starter/starter.go

```diff
@@ -717,12 +720,20 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 				gpm,
 				sm,
 				timeWatcher,
-				cfg,
+				tcfg,
 			)
 			if err != nil {
 				glog.Errorf("Error setting up PM recipient: %v", err)
 				return
 			}
+			mfv, _ := new(big.Int).SetString(*cfg.MaxFaceValue, 10)
+			if mfv == nil {
+				glog.Errorf("-maxFaceValue must be a valid integer, but %v provided. Restart the node with a different valid value for -maxFaceValue", *cfg.MaxFaceValue)
+				return
```

Comment:
```suggestion
				panic(fmt.Errorf("-maxFaceValue must be a valid integer, but %v provided. Restart the node with a different valid value for -maxFaceValue", *cfg.MaxFaceValue))
```

Please use `panic()` instead.

Reply:
Updated